lib/ClangImporter/ClangImporter.cpp

```diff
@@ -8316,3 +8316,19 @@ AccessLevel importer::convertClangAccess(clang::AccessSpecifier access) {
     return AccessLevel::Public;
   }
 }
+
+SmallVector<std::pair<StringRef, clang::SourceLocation>, 1>
+importer::getPrivateFileIDAttrs(const clang::Decl *decl) {
+  llvm::SmallVector<std::pair<StringRef, clang::SourceLocation>, 1> files;
+
+  auto prefix = StringRef("private_fileid:");
+
+  if (decl->hasAttrs())
```

Comment:
It's a very small stylistic thing, but could you put some braces for the outermost "then" statement?